test/parallel/test-stream3-pause-then-read.js

```diff
@@ -42,7 +42,7 @@ function read100() {
 function readn(n, then) {
   console.error('read %d', n);
   expectEndingData -= n;
-  ;(function read() {
+  (function read() {
```

Comment:
We enforce semicolons on the end of each expression, so this one seems unnecessary. Personally, I like using such IIFS to force a scope in ES5.